fluent-plugin-influxdb.gemspec

```diff
@@ -3,7 +3,7 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name          = "fluent-plugin-influxdb"
-  s.version       = '0.1.8'
```

Comment:
Could you remove this change?
I want to separate version up commit from feature implementation.